### PR TITLE
[RFC] tests: fix the glob tests when building under a long path

### PR DESCRIPTION
**Pull request by jszakmeister**

### CODE REVIEW

**Comment by tarruda** [comment]
:+1:

File: test/functional/eval/glob_spec.lua [hunk 1]
```diff
@@ -1,10 +1,15 @@
 local helpers = require('test.functional.helpers')
-local clear, execute, eval, eq = helpers.clear, helpers.execute, helpers.eval, helpers.eq
+local clear, execute, eval = helpers.clear, helpers.execute, helpers.eval
+local eq, feed = helpers.eq, helpers.feed
 
 before_each(function()
   clear()
   lfs.mkdir('test-glob')
   execute('cd test-glob')
```
Comment (fdinoff) [inline test/functional/eval/glob_spec.lua]: `execute('silent cd test-glob')`?

Comment (jszakmeister) [inline test/functional/eval/glob_spec.lua]: Good suggestion.  I like that idea better.  Will fix up.

**Comment by tarruda** [comment]
@jszakmeister is it possible to fix some of the quickbuild tap-parsing errors by sending warn/error messages to stderr? For example, this one sends the indeterminism warning:

```diff
diff --git a/test/functional/ui/screen.lua b/test/functional/ui/screen.lua
index aba6307..c427da7 100644
--- a/test/functional/ui/screen.lua
+++ b/test/functional/ui/screen.lua
@@ -254,7 +254,7 @@ function Screen:wait(check, timeout)
   end

   if failure_after_success then
-    print([[
+    io.stderr:write([[
 Warning: Screen changes have been received after the expected state was seen.
 This is probably due to an indeterminism in the test. Try adding
 `wait()` (or even a separate `screen:expect(...)`) at a point of possible
```

**Comment by justinmk** [comment]
Merged. Thanks for getting QB back on its feet!

**Comment by jszakmeister** [comment]
@tarruda It was the junit stuff that was a problem--the extra cruft would malform the XML.  With TAP, this should be a non-issue.

**Pull request closed without merging**